site/how.xml

```diff
@@ -210,74 +210,28 @@ limitations under the License.
-    </li>
-    <li>
-      <a href="https://github.com/ask/rabbitmq-munin">Munin Plugins for RabbitMQ</a><br />
-      Plugins for monitoring a RabbitMQ server with <a href="http://munin-monitoring.org">Munin</a>.
-    </li>
-    <li>
-      <a href="https://github.com/tnc/rac">RabbitMQ Administration Console </a><br />
-      RAC is an administration console for RabbitMQ written in PHP. (See <a href="http://twitpic.com/1mvo8o">this screenshot</a>).
-    </li>
-    <li>
-      <a href="https://github.com/auser/alice">Alice</a> and
-      <a href="https://github.com/auser/wonderland">Wonderland</a><br />
-      Provide RESTful administration and monitoring of RabbitMQ nodes and a
-      WebUI. Please read the <a href="http://willcodeforfoo.com/2009/07/13/announcing-alice">release notes</a> first.
-    </li>
-    <li>
-      <a href="https://github.com/opscode/cookbooks/tree/master/rabbitmq">Chef Cookbook</a><br />
-      A cookbook for managing RabbitMQ with Chef.
-    </li>
-    <li>
-      <a href="http://blog.thomasvandoren.com/monitoring-rabbitmq-queues-with-zabbix.html">Monitoring RabbitMQ with Zabbix</a><br/>
-      A simple example of how to configure Zabbix (an open source monitoring and
-      graphing tool) to monitor queue and exchange statistics in RabbitMQ.
-    </li>
-    <li>
-      <a href="https://github.com/paukul/humpty">Humpty</a><br />
-      A Sinatra/Ruby application to administer RabbitMQ over REST (via <a href="https://github.com/auser/alice">alice</a>)
-      and AMQP.
-    </li>
-    <li>
-      <a href="http://www.youtube.com/watch?v=CAak2ayFcV0">Monitoring RabbitMQ with Zenoss</a><br />
-      A video showing you how to monitor RabbitMQ with <a href="http://community.zenoss.org/">Zenoss</a>
-    </li>
-    <li>
-      <a href="http://community.zenoss.org/docs/DOC-5817">AMQP Event Monitor on Zenoss</a><br />
-      A daemon for monitoring AMQP with <a href="http://community.zenoss.org/">Zenoss</a>
-    </li>
-    <li>
-      <a href="https://github.com/gmr/Cony">Cony</a><br />
-      A HTTP based service written in Python for providing insight into running
-      RabbitMQ processes.
-    </li>
-    <li>
-      <a href="https://github.com/aaronfeng/trixx">Trixx</a><br />
-      An administration and monitoring tool written in Clojure so available for all
-      languages that run on JVM.
-    </li>
-    <li>
-      <a href="http://del.icio.us/alexisrichardson/rabbitmq+management">RabbitMQ Management</a> and
-      <a href="http://del.icio.us/alexisrichardson/rabbitmq+monitoring">RabbitMQ Monitoring</a> on del.icio.us<br />
-      More links to what others are doing to manage and monitor RabbitMQ.
-    </li>
+        <li>
+          <a href="https://github.com/rabbitmq/chef-cookbook">Chef Cookbook</a><br />
+          A cookbook for managing RabbitMQ with Chef.
+        </li>
+        <li>
+          <a href="https://forge.puppet.com/puppetlabs/rabbitmq">Puppet Module</a><br />
```

Comment:
This one has been deprecated last week, new one is: https://forge.puppet.com/puppet/rabbitmq